src/sa/sa_get_json.php

```diff
@@ -55,5 +55,49 @@ function run(Contact $user, $qreq, $ssel) {
     }
 }
 
-SearchAction::register("get", "json", SiteLoader::API_GET | SiteLoader::API_PAPER, new GetJson_SearchAction(false));
-SearchAction::register("get", "jsonattach", SiteLoader::API_GET | SiteLoader::API_PAPER, new GetJson_SearchAction(true));
+class GetJsonReviews_SearchAction extends SearchAction {
+    # tailored for Review Quality Collector. Don't remove parts.
+    private $iszip;
+    private $zipdoc;
+    function allow(Contact $user) {
+        return $user->is_manager();
+    }
+    function list_actions(Contact $user, $qreq, PaperList $pl, &$actions) {
+        $actions[] = [3070, $this->subname, "Reviews", "Reviews (JSON)"];
+    }
+    function run(Contact $user, $qreq, $ssel) {
+        $conf = $user->conf;
+        $results = [];
+        $results["hotcrp_version"] = HOTCRP_VERSION;
```

Comment:
It's important to filter the selection to only those papers that the user can in fact manage. `$user->can_administer($prow)`